### ISSUE TITLE
Empty the log at each game start.

### ISSUE THREAD

**Issue by RainerHilmer** [crosire/scripthookvdotnet#103]
It just grows and grows.

**Comment by JohnnyCrazy** [comment]
Mh...I don't know if it's a good idea deleting it every game-start since I (and propably a lot of other people?) often look older log-entries to see whats wrong exactly

Thoughts @crosire ?

**Comment by crosire** [comment]
Something like:
ScriptHookVDotNet-2015-05-20.log
ScriptHookVDotNet-2015-05-21.log
...?

**Comment by JohnnyCrazy** [comment]
Yea, but in a seperate folder
````
dontNetLogs/ScriptHookVDotNet-2015-05-20.log
dontNetLogs/ScriptHookVDotNet-2015-05-21.log
````

**Comment by IncoCode** [comment]
for N last days

**Comment by crosire** [comment]
I wouldn't create yet another folder actually. Just limit the number of days like @IncoCode said and once that limit is reached, delete the oldest file.

**Comment by JohnnyCrazy** [comment]
Well, IMO there are already a lot of files in the main directory and even more would just be more confusing. And since they are just interesting for developers (at least they should), a folder would be the best choice, but just my opinion :stuck_out_tongue_winking_eye:

**Comment by JohnnyCrazy** [comment]
Agree with crosire after a small conversation :+1:

**Comment by nevadascout** [comment]
Alternatively, could a "break" line be added whenever the game is launched?

Eg, 80 `#` chars, or two/three blank lines?

**Comment by Frede1012** [comment]
@nevadascout Only first time the game is launched, or also everytime the scripthook is reloaded? (I seem to like the latter better)

**Comment by nevadascout** [comment]
> everytime the scripthook is reloaded

That's a good idea actually. I like that as well

**Comment by RainerHilmer** [comment]
I am with Frede on this. It's less error prone for debugging.

**Comment by crosire** [comment]
Easier to implement too.

**Comment by Frede1012** [comment]
Yep ^

**Comment by crosire** [comment]
Implemented in c7dafc5ed4c9fc7ad123d6d38fba78f5b8786843 and follow up commit.
